### PR TITLE
Delete unnecessary RAW_EXTRACT logic

### PR DESCRIPTION
**Pull request by fejta**
ref https://github.com/kubernetes/test-infra/issues/2829
/assign @krzyzacy 
Update `--extract` test to send args to validate arg definition.

### CODE REVIEW

File: scenarios/kubernetes_e2e.py [hunk 1]
```diff
@@ -417,7 +408,8 @@ def main(args):
     if args.skew:
         runner_args.append('--skew')
 
-    setup_extract(args.extract, mode, runner_args)
+    for ext in args.extract or []:
+        runner_args.append('--extract=%s' % ext)
```
Comment (krzyzacy) [inline scenarios/kubernetes_e2e.py]: raise if --extract not exist?

**Comment by fejta** [comment]
Thanks

File: scenarios/kubernetes_e2e.py (hunk 1 above)
Comment (fejta) [inline scenarios/kubernetes_e2e.py]: We have a unit test that validates that all kubernetes_e2e.py scenarios add an `--extract` flag. I don't think we need to enforce that here.

Soon I'd actually like to move all of these flags out of kubernetes_e2e.py and into config.json

**Pull request merged**